packages/grafana-ui/src/components/Table/TableCellBuilder.tsx

```diff
@@ -157,10 +157,7 @@ class CellBuilderWithStyle {
     private column: Field,
     private replaceVariables: InterpolateFunction,
     private fmt?: ValueFormatter
-  ) {
-    //
-    console.log('COLUMN', column.name, theme);
-  }
+  ) {}
```

Comment:
Let's remove this empty constructor

Reply:
Can't remove it because it sets all the private properties that get used below
```
    private mapper: ValueMapper,
    private style: ColumnStyle,
    private theme: GrafanaTheme,
    private column: Field,
    private replaceVariables: InterpolateFunction,
    private fmt?: ValueFormatter
```

Reply:
Oh, I didn't know you can set properties this way, that's very cool!